### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Reporting a security vulnerability
```
Comment (dtantsur) [inline SECURITY_CONTACTS:1]: We should exclude this file to avoid confusion.

File: apis/go.mod [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-	github.com/google/uuid v1.1.2
-	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.0
+	github.com/google/uuid v1.3.0
+	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.2
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
-	sigs.k8s.io/controller-runtime v0.12.3
+	k8s.io/api v0.25.0
```
Comment (dtantsur) [inline apis/go.mod:10]: Ugh, this conflicts with the CVE fix :( Isn't it a bit too early to pull 0.25 bits?

File: SECURITY_CONTACTS (hunk 1 above)
Comment (honza) [reply SECURITY_CONTACTS:1]: Addressed in #253

**Pull request merged**